code/data_structures/src/list/singly_linked_list/operations/PalindromeInASinglyLinkedList.c

```diff
@@ -0,0 +1,63 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct node
+{
+    int  info;
+    struct node  *link;

```

Comment:
Inconsistency with `*`. Should be `struct node* link`, unless the style guide says otherwise.